src/storage/mongodb/TenantStorage.ts

```diff
@@ -190,10 +190,11 @@ export default class TenantStorage {
         $addFields: {
           logo: {
             $concat: [
-              `${Utils.buildRestServerURL()}/client/util/TenantLogo?ID=`,
+              `${Utils.buildRestServerURL()}/v1/util/tenants/logo?ID=`,
               { $toString: '$_id' },
-              '&LastChangedOn=',
-              { $toString: '$lastChangedOn' }
+              {
+                $ifNull: [{ $concat: ['&LastChangedOn=', { $toString: '$lastChangedOn' }] }, ''] // Only concat 'lastChangedOn' if not null
+              }
```

Comment:
Ok I got it.
Why not do the contrary, ensure that the tenant has always a lastChangedOn when changed at lest once?
If not, use the createdOn?

Reply:
> ensure that the tenant has always a lastChangedOn when changed at lest once?

Isn't it already how it works ?

I don't really get the purpose of adding the 'createdOn' in the request : if the tenant has not been updated, then it is not necessary to force the browser cache reload with an extra property, right ?

Reply:
> Isn't it already how it works ?

Do not ask me the question, check it ;-)

> I don't really get the purpose of adding the 'createdOn' in the request : if the tenant has not been updated, then it is not necessary to force the browser cache reload with an extra property, right ?

I agree, just check that the logo is reloaded from the browser's on a second call when lastChanged is not provided.
You can also check the Cars, CarCatalogs (master tenant)
Thansk!

Reply:
Done